backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/mapper/SubscriptionMapper.java

```diff
@@ -32,4 +35,8 @@ public default List<SubscriptionEntity> toEntity(@Nullable Iterable<Subscription
 	@Mapping(target = "isNew", ignore = true)
 	SubscriptionEntity toEntity(@Nullable Subscription subscription);
 
+	@Mapping(target = "isNew", ignore = true)
+	@Mapping(target = "alertType", source = "alertType")
+	@BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
+	SubscriptionEntity update(@Nullable Subscription subscription, @MappingTarget SubscriptionEntity target);	
```

Comment:
Since the input is `@Nullable`, the return value should also be annotated as `@Nullable`.

MapStruct will return a null if the input is null. You can check this by looking at the mapper implementation:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48123208/e29b5733-e9f5-4354-843f-c20514540be7)

Reply:
added